### DISCUSSION TITLE
Recomended Upgrade Path

### DISCUSSION THREAD

**Discussion by whitepiratebaku** [minio/minio#18048] [Q&A] [answered]
Is there recommended path for upgrades which described on docs? I understand there is release notes and they should be read. But imagine a really old server like with version 2020... when you wanna upgrade probably it is not ideal to go the latest directly but there should be some path to follow.

**Comment by harshavardhana** [comment, marked as answer]
@whitepiratebaku we recommend that you upgrade to latest release there are no special upgrade paths.

Just download the binary replace the current binary and then `mc admin service restart`

Please make sure you do this in a QA or staging environment first that's all.
